05week/checkers.js

```diff
@@ -61,7 +117,61 @@ class Game {
   }
   start() {
     this.board.createGrid();
+    this.board.addCheckers();
   }
+  moveChecker(whichPiece, toWhere) {
```

Comment:
Very nice thought process! For ease of readability, think of a way to refactor this into a smaller set of functions/methods.